src/ports/postgres/modules/deep_learning/madlib_keras_model_selection.sql_in

```diff
@@ -873,9 +872,9 @@ CREATE OR REPLACE FUNCTION MADLIB_SCHEMA.generate_model_configs(
     object_table            VARCHAR DEFAULT NULL
 ) RETURNS VOID AS $$
     PythonFunctionBodyOnly(`deep_learning', `madlib_keras_model_selection')
-    with AOControl(False):
```

Comment:
If the user has appendonly set to true, then our code will fail if we don't have the `with aocontrol(False)` code. 
You can reproduce this by setting `set gp_default_storage_options='appendonly=true';` and then running load_model_selection_table/generate_model_configs will error out with
```
ERROR:  spiexceptions.FeatureNotSupported: append-only tables do not support unique indexes
```